locale/ru/LC_MESSAGES/Structural/README.po

```diff
@@ -1,19 +1,21 @@
-# 
+#
 msgid ""
 msgstr ""
 "Project-Id-Version: DesignPatternsPHP 1.0\n"
 "Report-Msgid-Bugs-To: \n"
 "POT-Creation-Date: 2015-05-29 12:18+0200\n"
-"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
-"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
-"Language-Team: LANGUAGE <LL@li.org>\n"
+"PO-Revision-Date: 2015-05-30 23:27+0300\n"
+"Last-Translator: Eugene Glotov <kivagant@gmail.com>\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
+"Language: ru\n"
 
 #: ../../Structural/README.rst:2
 msgid "Structural"
 msgstr ""
+"`Структурные шаблоны проектирования <https://ru.wikipedia.org/wiki/"
+"Структурные_шаблоны_проектирования>` (`Structural`__)"
```

Comment:
ok i'll add these english links in README.rst
    Behavioral pattern
    Creational pattern
    Structural pattern

```
 #: ../../Structural/README.rst:2
 msgid "`Structural`__"
 msgstr ""
"`Структурные шаблоны проектирования <https://ru.wikipedia.org/wiki/Структурные_шаблоны_проектирования>`_ (`Structural`__)"
```